packages/hash/api/src/auth/ory-kratos.ts

```diff
@@ -1,10 +1,23 @@
-import { Configuration } from "@ory/client";
+import { Configuration, Identity } from "@ory/client";
 import { V0alpha2Api as OpenSourceV0alpha2Api } from "@ory/kratos-client";
 import { getRequiredEnv } from "../util";
 
-/** @todo: connect to admin endpoint when admin methods are required */
 const publicUrl = getRequiredEnv("ORY_KRATOS_PUBLIC_URL");
 
-export const kratosSdk = new OpenSourceV0alpha2Api(
+export const publicKratosSdk = new OpenSourceV0alpha2Api(
   new Configuration({ basePath: publicUrl }),
 );
+
+const adminUrl = getRequiredEnv("ORY_KRATOS_ADMIN_URL");
+
+export const adminKratosSdk = new OpenSourceV0alpha2Api(
+  new Configuration({ basePath: adminUrl }),
+);
```

Comment:
Previously I expected to be able to use the same client for both admin and public API calls to ory kratos. However whenever I called public endpoints on the admin URL I got authentication errors. I couldn't find kratos documentation explaining why this is the case, and whether there's a way to get around this.

Splitting the clients into two works (one for public calls, one for admin calls, each initialised with the separate URLs), so that's what I've gone with for now.